recipes/stoatydive/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "StoatyDive" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "aab0c2480e920b2d7129cccef91e71c07f97b2f3c5ce8526c23b26536d44dc79" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/heylf/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    noarch: python
+
+requirements:
+    host:
+        - python
+    run:
+        - python
+        - bedtools >=2.27.1
+        - numpy
+        - matplotlib
+        - scipy =1.2
```

Comment:
Is scipy 1.3 not supported for some reason?

Reply:
Thanks for the remark. I will make a fix than it should work with higher versions of scipy.

Reply:
Its fixed!